examples/mysql-call-stored-procedures/mysql_call_stored_procedures.bal

```diff
@@ -81,6 +81,11 @@ function insertRecord(mysql:Client sqlClient) {
     if (retCall is sql:ProcedureCallResult) {
         io:println("Call stored procedure `InsertStudent` is successful : ", 
                 retCall.executionResult);
+
+        error? e = retCall.close();
```

Comment:
```suggestion
        _ = retCall.close();
```
Update in other places as well

Reply:
Yes noted. I will update them. Thank you.